rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/nodesource/common/PluginDescriptor.java

```diff
@@ -138,6 +141,15 @@ private void findSectionDescriptions(Class<?> cls, Object instance) {
         }
     }
 
+    private void findMeta(Class<?> cls, Object instance) {
+        try {
+            Method getMetaMethod = cls.getMethod("getMeta");
+            Map<String, String> metaInfo = (Map<String, String>) getMetaMethod.invoke(instance);
+            this.meta.putAll(metaInfo);
+        } catch (NoSuchMethodException | InvocationTargetException | IllegalAccessException e) {
```

Comment:
Shouldn't one of these exception be at least be logged ? I bet they can contribute to debugging.